pkg/test/harness.go

```diff
@@ -299,6 +335,10 @@ func (h *Harness) Run() {
 		}
 	}
 
+	if err := h.DoKubectl(); err != nil {
```

Comment:
so we run the kubectl command before starting kudo? 🤔 (line 343)

Reply:
For the harness, yes. This allows us to install prerequisites for KUDO (e.g., CRDs). Commands defined in TestSteps run after everything has started up.